app/uk/gov/hmrc/helptosavefrontend/controllers/HelpToSaveAuth.scala

```diff
@@ -73,24 +70,28 @@ class HelpToSaveAuth(app: Application) extends FrontendController with Authorise
   def authorisedForHtsWithConfidence(action: HtsAction): Action[AnyContent] = {
     Action.async { implicit request =>
       authorised(AuthWithConfidence) {
-        action(request)
+        action(request)(HtsContext(None, isAuthorised = true))
       }.recover {
         case e ⇒ handleFailure(e)
       }
     }
   }
 
+  def isAuthorised(action: HtsAction): Action[AnyContent] = {
```

Comment:
`isAuthorised` is a bit misleading - it sounds like it should be returning a boolean, but what it's actually doing is performing the action whether or not the user is logged in. Perhaps naming it `unprotected` or something along those lines would be more descriptive and makes it clear that the action will still occur if the user is not logged in